ipywidgets/widgets/interaction.py

```diff
@@ -217,6 +217,11 @@ def call_f(name=None, old=None, new=None):
         if manual:
             manual_button.disabled = True
         try:
+            def close_result_widgets(w):
+                if type(w.result) is Box:
+                    close_result_widgets(w.result)
+                    w.result.close()
+            close_result_widgets(container)
```

Comment:
This won't work if people don't display their widgets within the interact container and I haven't seen anyone display additional widgets inside the interact container yet, so my guess is that this won't do anything for most people.